<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChatWindow</string>
  </property>
  <widget class="QTextBrowser" name="MessageBrowserBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextInputBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>981</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>530</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="TransferProgressBar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="TransferProgressText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>传输进度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="DownloadSpeedText">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>530</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0mb/s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FileSelectionButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>530</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
